marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/flow-utils.sjs

```diff
@@ -711,18 +705,8 @@ class FlowUtils {
   }
 
   parseText(text, outputFormat){
-    try {
```

Comment:
This is a good example of what happens when there are no tests - this function has multiple problems with it (datahub isn't defined, and 'instance' is used instead of 'text'), and it's used by custom ingestion step modules.